packages/backend/src/miro/parse-board-id.spec.ts

```
@@ -0,0 +1,75 @@
+import { BadRequestException } from '@nestjs/common';
+import { parseMiroBoardId } from './parse-board-id';
+
+describe('parseMiroBoardId', () => {
+  it('extracts the id from a canonical board URL', () => {
+    expect(parseMiroBoardId('https://miro.com/app/board/uXjVOD50NUI=/')).toBe('uXjVOD50NUI=');
```

ESLint

[error] Replace `'uXjVOD50NUI='` with `⏎······'uXjVOD50NUI=',⏎····`

(prettier/prettier)